docker-compose.yml

```diff
@@ -642,7 +642,5 @@ volumes:
 
 networks:
   default:
-    ipam:
-      driver: default
-      config:
-        - subnet: ${DOCKER_SUBNET:-192.168.21.0/24}
+    name: ${DOCKER_NETWORK_NAME}
```

Comment:
We could also just skip the whole network block, but the idea was to make this easily configurable as the default docker network actually conflicts with the ip range that is used in German ICE trains 🙈 

Maybe we can just document that option in case someone uses the same IP range (and check that in the bootstrap.sh script)

Reply:
Hmmm... Do you mean we should delete the network section in the docker compose ?

I understand the problem you have with the German ICE trains ^^
We also have some problems with the trains in France ^^'

> Maybe we can just document that option in case someone uses the same IP range (and check that in the bootstrap.sh script)

Normally, with the `docker network create` command, it creates a new network with an IP range that is available.

```bash
function init_subnet_docker() {
	uuidgen="$(uuidgen)"
	idNetwork="$(docker network create -d bridge nextcloud_network_"${uuidgen}")"
	networkName="$(docker network inspect "${idNetwork}" | jq .[].Name | tr -d '"')"
	echo "$networkName"
}
```

We could write a section on this in the documentation to prevent the user.
But, I don't think it's an option. Either an IP address range is created or it is not, I think.

Or, maybe Didn't I understand you ? ^^'